src/EfficientnetModel.ts

```diff
@@ -86,22 +92,42 @@ export default class EfficientNetModel {
 
   private async predict(
     tensor: tf.Tensor3D,
-    topK: number
+    topK: number,
+    locale: string
   ): Promise<EfficientNetResult> {
     const objectArray = this.model!.predict(tensor) as tf.Tensor;
     const values = objectArray.dataSync() as Float32Array;
-    return new EfficientNetResult(values, topK);
+    return new EfficientNetResult(values, topK, locale);
   }
 
   async inference(
-    imgPath: string | Buffer,
+    options: EfficientNetModelInferenceOptions
+  ): Promise<EfficientNetResult>;
+  async inference(
+    options: string | Buffer,
+    topK?: number
+  ): Promise<EfficientNetResult>;
+  async inference(
```

Comment:
please change the signture to 

async inference(
    imagePath: string | Buffer,
    options: EfficientNetModelInferenceOptions
  ): Promise<EfficientNetResult>;

Reply:
Don't consider forward compatibility?

Or, `options: EfficientNetModelInferenceOptions | number`, `typeof options === 'number'` exec different logic

Reply:
for inference, image is always a required parameter all the rest should be in options

Reply:
I mean, if someone use `node-efficientnet` such as `mode.inference(img, 5)`, we make second param to be `EfficientNetModelInferenceOptions `,  previous code will report an error